avaframe/runCom2AB.py

```diff
@@ -2,44 +2,74 @@
 """
 
 import logging
+import logging.config
 import sys
 import os
+import configparser
 
 # Local imports
 from avaframe.com2AB import com2AB
-import avaframe.com2AB.com2ABCfg as conf
 from avaframe.out3SimpPlot import outAB
+from avaframe.configLogger import ConfigLogger
 
-# create logger, set to logging.DEBUG to see all messages
-# logging.basicConfig(filename='com2AB.log', filemode='w', level=logging.INFO,
-#                     format='%(module)s:%(levelname)s - %(message)s')
-# logging.basicConfig(stream=sys.stdout, level=logging.INFO,
-#                     format='%(module)s:%(levelname)s - %(message)s')
-logFileName = os.path.join(conf.saveOutPath, "runCom2AB.log")
-logging.basicConfig(
-    level=logging.INFO,
-    format="%(module)s [%(levelname)s] %(message)s",
-    handlers=[
-        logging.FileHandler(logFileName, "w"),
-        logging.StreamHandler()
-    ]
-)
+#---------------------------------------------
+#---------------------------------------------
+#############################################
+# Load all input Parameters from congig file #
+#############################################
+
+cfg = configparser.ConfigParser(allow_no_value=True)
+if os.path.isfile('avaframe/com2AB/local_com2ABCfg.ini'):
+    cfg.read('avaframe/com2AB/local_com2ABCfg.ini')
+else:
+    cfg.read('avaframe/com2AB/com2ABCfg.ini')
+
+cfgPath = cfg['INPATH']
+cfgsetup = cfg['ABSETUP']
+cfgFlags = cfg['FLAGS']
+
+#########################################
+# Load all input Parameters for logging #
+#########################################
+logFileName = os.path.join(cfgPath['saveOutPath'], "runCom2AB.log")
+if os.path.exists(logFileName):
+    os.remove(logFileName)
+logging.config.fileConfig(fname='avaframe/logging.conf', defaults={'logfilename': logFileName}, disable_existing_loggers=False)
 log = logging.getLogger(__name__)
+config_logger = ConfigLogger(log)
+config_logger(cfg)
```

Comment:
This could be moved to configLogger.py, so one would just have to call
log = configLogger.initiateConfig(logFileName) 
and
configLogger.logCfg(cfg)